packages/web/src/js/FileUploader.ts

```diff
@@ -23,8 +24,10 @@ class FileUploader extends BaseComponent {
   isDragAndDropSupported: boolean;
   fileSizeLimit: number;
   fileQueueLimit: number;
+  isDismissible: boolean;
   inputName: string;
   isMultiple: boolean;
+  accept: string;
```

Comment:
I think the object properties should be listed alphabetically. (ESlint, can you hear me…? 🙉)

Reply:
https://www.npmjs.com/package/eslint-plugin-sort-class-members

Reply:
I have filed an issue for this.